book/src/snapshot-restore-feature.md

```diff
@@ -2,10 +2,11 @@
 
 ## Status
 
-Status | Min K8s Version | Max K8s Version | external-snapshotter Version | external-provisioner Version
---|--|--|--|--
-Alpha | 1.12 | 1.12 | 0.4 | 0.4
-Alpha | 1.13 | - | 1.0+ | 1.0+
+Status | Min K8s Version | Max K8s Version | snapshot-controller Version | CSI external-snapshotter sidecar Version | external-provisioner Version
+--|--|--|--|--|--
+Alpha | 1.12 | 1.12 | | 0.4+ | 0.4+
+Alpha | 1.13 | 1.16 | | 1.0+ | 1.0+
+Beta  | 1.17 | - | 2.0+ | 2.0+ | 1.5+
```

Comment:
Should we remove this table from this section?  This is redundant and also not consistent with the table in external-snapshotter.md.